### PR TITLE
chore: updating api version for plug & play release

### PR DESCRIPTION
**Pull request by jebrando**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: service/src/twin.ts [hunk 1]
```diff
@@ -56,6 +56,10 @@ export class Twin implements TwinData {
    * Tag used in optimistic concurrency to avoid multiple parallel editions of the device twin.
    */
   etag: string;
+  /**
+   * Model Id of the device identity associated with the twin, as it exists in the device identity registry.
+   */
+  modelId?: string;
   /**
```
Comment (anthonyvercolano) [inline service/src/twin.ts:63]: How the heck did I miss this.

File: service/test/_twin_test.js [hunk 2]
```diff
@@ -91,9 +91,11 @@ describe('Twin', function() {
     it('calls the getModuleTwin method on the Registry', function() {
       var fakeDeviceId = 'deviceId';
       var fakeModuleId = 'moduleId';
+      var fakeModelId = 'modelId';
```
Comment (anthonyvercolano) [inline service/test/_twin_test.js:94]: >var fakeModelId = 'modelId'; [](start = 6, length = 28)

Shouldn't this be on regular twin also?

File: service/src/twin.ts (hunk 1 above)
Comment (YoDaMa) [reply service/src/twin.ts:63]: genius

File: service/test/_twin_test.js [hunk 3]
```diff
@@ -91,9 +91,11 @@ describe('Twin', function() {
...
       var registry = new Registry(fakeConfig, {});
       var twin = new Twin(fakeDeviceId, registry);
       twin.moduleId = fakeModuleId;
+      twin.modelId = fakeModelId;
```
Comment (anthonyvercolano) [inline service/test/_twin_test.js:98]: >twin.modelId = fakeModelId; [](start = 6, length = 27)

I don't see any assertion based on this.

**Review by anthonyvercolano — APPROVED** [review]
:shipit:

**Comment by zolvarga** [comment]
>  */

Rename this file to "_digital_twin_client_test.js"

---
Refers to: service/test/_digital_twin_service_client_test.js:6 in 1ccaefb. [](commit_id = 1ccaefb988ff35dc3a2e89f2518d3bcde887ef43, deletion_comment = False)

File: service/samples/typescript/get_digital_twins.ts [hunk 4]
```diff
@@ -1,11 +1,11 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-import { DigitalTwinServiceClient } from 'azure-iothub';
+import { DigitalTwinClient } from 'azure-iothub';
 import { IoTHubTokenCredentials } from 'azure-iothub';
 
 // Simple example of how to:
-// - create a Digital Twin Service Client using the DigitalTwinServiceClient constructor
+// - create a Digital Twin Service Client using the DigitalTwinClient constructor
```
Comment (zolvarga) [inline service/samples/typescript/get_digital_twins.ts:8]: >Digital Twin Service [](start = 14, length = 20)

Rename those too

File: service/samples/typescript/invoke_command.ts [hunk 5]
```diff
@@ -10,7 +10,7 @@ const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnOn'; // for the envi
 const commandPayload = process.env.IOTHUB_COMMAND_PAYLOAD || ''; // for the environmental sensor, it really doesn't matter. any string will do.
 
 // Simple example of how to:
-// - create a Digital Twin Service Client using the DigitalTwinServiceClient constructor
+// - create a Digital Twin Service Client using the DigitalTwinClient constructor
```
Comment (zolvarga) [inline service/samples/typescript/invoke_command.ts:13]: >Digital Twin Service Clien [](start = 14, length = 26)

Rename...

File: service/samples/typescript/invoke_component_command.ts [hunk 6]
```diff
@@ -11,7 +11,7 @@ const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnOn'; // for the envi
 const commandPayload = process.env.IOTHUB_COMMAND_PAYLOAD || ''; // for the environmental sensor, it really doesn't matter. any string will do.
 
 // Simple example of how to:
-// - create a Digital Twin Service Client using the DigitalTwinServiceClient constructor
+// - create a Digital Twin Service Client using the DigitalTwinClient constructor
```
Comment (zolvarga) [inline service/samples/typescript/invoke_component_command.ts:14]: >Digital Twin Service Client [](start = 14, length = 27)

Rename

File: service/samples/javascript/get_digital_twin.js [hunk 7]
```diff
@@ -4,11 +4,11 @@
 'use strict';
 
 const IoTHubTokenCredentials = require('azure-iothub').IoTHubTokenCredentials;
-const DigitalTwinServiceClient = require('azure-iothub').DigitalTwinServiceClient;
+const DigitalTwinClient = require('azure-iothub').DigitalTwinClient;
 const { inspect } = require('util');
 
 // Simple example of how to:
-// - create a Digital Twin Service Client using the DigitalTwinServiceClient constructor
+// - create a Digital Twin Service Client using the DigitalTwinClient constructor
```
Comment (zolvarga) [inline service/samples/javascript/get_digital_twin.js:11]: >Digital Twin Service Client [](start = 14, length = 27)

Rename

File: service/samples/javascript/invoke_command.js [hunk 8]
```diff
@@ -4,11 +4,11 @@
 'use strict';
 
 const IoTHubTokenCredentials = require('azure-iothub').IoTHubTokenCredentials;
-const DigitalTwinServiceClient = require('azure-iothub').DigitalTwinServiceClient;
+const DigitalTwinClient = require('azure-iothub').DigitalTwinClient;
 const { inspect } = require('util');
 
 // Simple example of how to:
-// - create a Digital Twin Service Client using the DigitalTwinServiceClient constructor
+// - create a Digital Twin Service Client using the DigitalTwinClient constructor
```
Comment (zolvarga) [inline service/samples/javascript/invoke_command.js:11]: >Digital Twin Service Client [](start = 14, length = 27)

Rename

File: service/samples/javascript/invoke_component_command.js [hunk 9]
```diff
@@ -4,11 +4,11 @@
 'use strict';
 
 const IoTHubTokenCredentials = require('azure-iothub').IoTHubTokenCredentials;
-const DigitalTwinServiceClient = require('azure-iothub').DigitalTwinServiceClient;
+const DigitalTwinClient = require('azure-iothub').DigitalTwinClient;
 const { inspect } = require('util');
 
 // Simple example of how to:
-// - create a Digital Twin Service Client using the DigitalTwinServiceClient constructor
+// - create a Digital Twin Service Client using the DigitalTwinClient constructor
```
Comment (zolvarga) [inline service/samples/javascript/invoke_component_command.js:11]: >Digital Twin Service Client [](start = 14, length = 27)

Rename

File: service/samples/javascript/update_digital_twin.js [hunk 10]
```diff
@@ -4,10 +4,10 @@
 'use strict';
 
 const IoTHubTokenCredentials = require('azure-iothub').IoTHubTokenCredentials;
-const DigitalTwinServiceClient = require('azure-iothub').DigitalTwinServiceClient;
+const DigitalTwinClient = require('azure-iothub').DigitalTwinClient;
 
 // Simple example of how to:
-// - create a Digital Twin Service Client using the DigitalTwinServiceClient constructor
+// - create a Digital Twin Service Client using the DigitalTwinClient constructor
```
Comment (zolvarga) [inline service/samples/javascript/update_digital_twin.js:10]: >Digital Twin Service Client [](start = 14, length = 27)

Rename

File: service/devdoc/service_client_requirements.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Digital Twin Service Client requirements
```
Comment (zolvarga) [inline service/devdoc/service_client_requirements.md:1]: >Digital Twin Service Client [](start = 2, length = 27)

Rename

File: service/test/_twin_test.js [hunk 12]
```diff
@@ -143,7 +143,7 @@ describe('Twin', function() {
     /*Tests_SRS_NODE_IOTHUB_TWIN_16_023: [The method shall call the `done` callback with a `null` error object, its parent instance as a second argument and the transport `response` object as a third argument if the request succeeds**/
     it('calls the done callback with a null error object, a twin and a response', function(testCallback) {
       var fakeDeviceId = 'deviceId';
-      var fakeTwin = { deviceId: fakeDeviceId };
+      var fakeTwin = { deviceId: 'fakeDeviceId' };
```
Comment (anthonyvercolano) [inline service/test/_twin_test.js:146]: >ar fakeTwin = { deviceId: 'fakeDeviceId' }; [](start = 7, length = 43)

This can't be right.

**Review by anthonyvercolano — CHANGES_REQUESTED** [review]
:clock1:

File: service/samples/typescript/invoke_command.ts (hunk 5 above)
Comment (jebrando) [reply service/samples/typescript/invoke_command.ts]: Done

File: service/samples/typescript/get_digital_twins.ts (hunk 4 above)
Comment (jebrando) [reply service/samples/typescript/get_digital_twins.ts]: Done

**Review by zolvarga — APPROVED** [review]
:shipit:

File: service/samples/javascript/update_digital_twin.js (hunk 10 above)
Comment (jebrando) [reply service/samples/javascript/update_digital_twin.js]: Done

File: service/devdoc/service_client_requirements.md (hunk 11 above)
Comment (jebrando) [reply service/devdoc/service_client_requirements.md]: Done

File: service/test/_twin_test.js (hunk 12 above)
Comment (jebrando) [reply service/test/_twin_test.js:146]: You sir are correct.

**Review by anthonyvercolano — APPROVED** [review]
:shipit:

**Pull request merged**